apps/usecase/list_user_orders.py

```
@@ -0,0 +1,44 @@
+            has_more = query.page * query.limit < total
+            return {
+                "orders": summaries,
+                "pagination": {
+                    "page": query.page,
+                    "limit": query.limit,
+                    "total": total,
+                    "has_more": has_more
+                }
+            }, 200
+
+        except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)